bin/improver-weighted-blending

```diff
@@ -123,13 +128,24 @@ def main():
 
```

Comment:
The way I've it set up now, the error calling relies on args.cval or args.slope being None or not. I've added a comment to hopefully clarify this. Hopefully #139 will come up with a better solution to this.